examples/physics/run_fmm.py

```
@@ -1098,7 +1098,7 @@ def tensor_collect_norm(d):
     elif isinstance(d, shamrock.math.SymTensorCollection_f64_1_1):
         return np.sqrt(d.t1.inner(d.t1))
     else:
-        raise ValueError(f"Unsupported tensor collection type: {type(d)}")
+        raise TypeError(f"Unsupported tensor collection type: {type(d)}")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)